lib/saluki-event/src/metric/value/sketch.rs

```diff
@@ -188,6 +188,19 @@ impl<'a> IntoIterator for &'a SketchPoints {
     }
 }
 
+impl FromIterator<(Option<NonZeroU64>, DDSketch)> for SketchPoints {
+    fn from_iter<T: IntoIterator<Item = (Option<NonZeroU64>, DDSketch)>>(iter: T) -> Self {
+        let mut sketch_points = SketchPoints(TimestampedValues::default());
+        for (ts, sketch) in iter {
+            sketch_points
+                .0
+                .values
+                .push(TimestampedValue::from((ts.unwrap().get(), sketch)));
```

Comment:
We shouldn't be unwrapping if we're knowingly accepting `Option<T>` here.. so we should add a new `From` impl -- `impl<T> From<(Option<NonZeroU64>, T)> for TimestampedValue<T>` -- to let us convert properly.